frameworks/g-cloud-9/questions/services/webChatSupportAccessibility.yml

```diff
@@ -0,0 +1,24 @@
+name: "Web chat support accessibility"
+question: "Does your webchat comply with the Web Content Accessibility Guidelines 2.0?"
```

Comment:
Same thing as before - this need to be "To what level does your webchat comply..." - as it is this is a yes/no question.